versions/unversioned/guides/using-apollo.rst

```diff
@@ -0,0 +1,177 @@
+reason why GraphQL is interesting to so many developers is that it improves upon
+the failings of older API architectures like SOAP and REST.
+
+For instance, with a GraphQL query you are guaranteed to only ever get the
+*exact data you need*. No more overfetching or underfetching from API endpoints.
+Not only does a GraphQL query allow you to get exactly the data you need, but it
+also allows you to do so in one request as contrasted to REST APIs of the past
+which required many HTTP requests to assemble meaningful data for your UI.
+Combined with a static type system, GraphQL provides both a powerful user
+experience and developer experience.
+
+To learn more about GraphQL see `graphql.org <http://graphql.org/>`_.
+
+Why Apollo?
+^^^^^^^^^^^
+
+Apollo is not the only library designed to help you manage your API data.
+However, the principles which Apollo was designed around make it a powerful yet
+still flexible choice. Some of those principles are for Apollo to be:
+
+1. Incrementally adoptable.
+2. Universally compatible.
+3. Inspectable and understandable.
+4. Built for interactive apps.
+5. Community driven.
+
+The Apollo client does way more then run your queries. To learn more about the
+advantages of using Apollo client see
+`dev.apollodata.com/react <http://dev.apollodata.com/react/>`_.
+
+1. Installing Apollo Client
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To get started with Apollo and Exponent, install the
+`apollo-client <http://npmjs.com/apollo-client>`_ npm package, the
+`react-apollo <https://www.npmjs.com/package/react-apollo>`_ React integration
+package, and the `graphql-tag <https://www.npmjs.com/package/graphql-tag>`_
+library for constructing query documents.
+
+.. code-block:: bash
+
+   npm install apollo-client react-apollo graphql-tag --save
+
+These packages are modular building blocks that can be used independently in
+other environments.
+
+2. Client Setup
+^^^^^^^^^^^^^^^
+
+To start using Apollo we need to create an ``ApolloClient`` and serve that
+client to our React application with an ``ApolloProvider``. The ``ApolloClient``
+is what controls all your GraphQL data and the ``ApolloProvider`` wires the
+client into your React component hierarchy.
+
+Creating a client
+"""""""""""""""""
+
+Initialize an instance of the ``ApolloClient`` imported from the
+`apollo-client <http://npmjs.com/apollo-client>`_ package and provide the client
+with a ``networkInterface`` that points to your GraphQL API.
+
+.. code-block:: javascript
+
+    import ApolloClient, { createNetworkInterface } from 'apollo-client';
+
+    // Replace http://my-api.graphql.com with your GraphQL API’s URL.
+    const client = new ApolloClient({
+      networkInterface: createNetworkInterface({ uri: 'http://my-api.graphql.com' }),
+    });
+
+Creating a provider
+"""""""""""""""""""
+
+Next you will want to connect your client to the rest of your React app with
+``ApolloProvider`` from the
+`react-apollo <https://www.npmjs.com/package/react-apollo>`_ package.
```

Comment:
I think in today's world it's pretty safe to assume everyone using React is familiar with Redux, and the Exponent starter app has it AFAIK